core/server/api/shared/validators/input/all.js

```diff
@@ -50,7 +50,12 @@ const validate = (config, attrs) => {
                 });
 
                 if (unallowedValues.length) {
-                    errors.push(new common.errors.ValidationError());
+                    errors.push(new common.errors.ValidationError({
+                        message: common.i18n.t('notices.data.validation.index.validationFailed', {
+                            validationName: 'AllowedValues',
```

Comment:
Pretty open to changing validation name if `AllowedValues` doesn't seem right :)

Reply:
I think it's fine for now 👍